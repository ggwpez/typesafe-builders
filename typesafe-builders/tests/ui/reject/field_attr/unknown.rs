use typesafe_builders::prelude::*;

#[derive(Builder)]
struct Struct {
	#[builder(yikes)]
	x: u8,
}

fn main() { }
