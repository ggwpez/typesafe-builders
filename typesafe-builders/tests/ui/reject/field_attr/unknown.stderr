error: Unknown field attribute: "yikes"
 --> tests/ui/reject/field_attr/unknown.rs:5:12
  |
5 |     #[builder(yikes)]
  |               ^^^^^
